component/custom/custom_loader.go

```diff
@@ -0,0 +1,67 @@
+package custom
```

Comment:
As a common coding practice we use the same name in both file and directory so lets rename this file to custom.go